pom.xml

```diff
@@ -234,10 +247,27 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>git-server</artifactId>
+      <groupId>org.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <version>4.2.1</version>
       <scope>test</scope>
     </dependency>
+    <!--
+      only to get the slf4j jul not here, the dependency should be provided in jenkins-war
+      plugins should be able to use what they prefer and not force to an implementation
+    -->
+    <dependency>
+      <groupId>org.jenkins-ci.main</groupId>
+      <artifactId>jenkins-war</artifactId>
+      <type>executable-war</type>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
I agree with @jtnord that I don't see the benefit of the test dependency on slf4j-simple.
```suggestion
```

Reply:
because it's so simple to configure a file `simplelogger.properties` when some logs are needed.